lib/request.js

```diff
@@ -19,7 +28,10 @@ module.exports = function (protocol, host, version, token) {
             pathname: 'api/v1beta2/' + object.endpoint
         });
         **/
-        return protocol + '://' + host + '/api/' + version + '/' + object.endpoint;
+        if (object.options && object.options.apiPrefix){
```

Comment:
Allow use of an API prefix on a per-collection basis. (for example, OpenShift has the prefix `osapi` rather than the default `api`)